apps/builder/app/builder/features/props-panel/controls/combined.tsx

```diff
@@ -19,6 +19,10 @@ export const renderControl = ({
     throw Error("Data source is not resolved");
   }
 
+  if (prop?.type === "action") {
```

Comment:
We don't render control when its bound? maybe a comment for why/what

Reply:
we don't have ui for actions. That's all.

Reply:
Maybe good idea to have a todo with a link to an issue in such cases